README.md

```diff
@@ -39,7 +39,7 @@ If you're not calling `cargo` directly and cannot change how it's invoked, you c
 
 Microsoft uses `cargo auditable` internally and maintains the [data extraction library for Go](https://github.com/microsoft/go-rustaudit).
```

Comment:
```suggestion
Microsoft uses `cargo auditable` internally and maintains the [data extraction library for Go](https://github.com/microsoft/go-rustaudit).

Chainguard includes `cargo auditable` in their rust base container, with a default `cargo` wrapper to always call `cargo auditable`, so that Rust applications built using this container are auditable by default.
```

Reply:
This would be my suggestion, does that sound good to the two of you?